Inspicio/Extensions/IdentityExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿    
+    // Jack Lloyd [06/07/17]
+/*
+    Extension methods for the Identity. (ApplicationUser additional columns)
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Security.Principal;
+using System.Threading.Tasks;
+
+namespace Inspicio.Extensions
+{
+    public static class IdentityExtensions
+    {
+        public static string GetProfileName(this ClaimsPrincipal user)
+        {
+            // Jack Lloyd [06/07/17]
+            // ClaimTypes.GivenName returning the ProfileName
+            var profileName = user.Claims.FirstOrDefault(v => v.Type == ClaimTypes.GivenName).Value;
+
+            // Jack Lloyd [06/07/17]
+            // Do the null check here, will avoid having local issues elsewhere.
+            return (profileName != null) ? profileName : String.Empty;
```

Comment:
C# has a "[null coalescing operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operator)" which should let you simplify this down to 
```
public static string GetProfileName(this ClaimsPrincipal user)
{
    return user.Claims.FirstOrDefault(v => v.Type == ClaimTypes.GivenName).Value ?? string.Empty;
}
```